lib/defaults/defaults.go

```diff
@@ -512,6 +512,11 @@ var (
 	// the Teleport configuration file that tctl reads on use
 	ConfigFileEnvar = "TELEPORT_CONFIG_FILE"
 
+	// ProxyFallbackAddrEnvar optionally specifies the address to use as a fallback when
+	// dialing through reverse tunnel fails on first time connect.
+	// See https://github.com/gravitational/teleport/issues/4141 for details.
+	ProxyFallbackAddrEnvar = "TELEPORT_PROXY_FALLBACK_ADDR"
```

Comment:
Since it's no longer a fallback address, but an alternative address to use if specified, how about `TELEPORT_TUNNEL_PUBLIC_ADDR`?